pootle/core/management/subcommands.py

```diff
@@ -6,241 +6,21 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
-import os
-import sys
 from collections import OrderedDict
 
-from django.core.management import (
-    BaseCommand, CommandError, CommandParser, handle_default_options)
-from django.core.management.base import SystemCheckError
-from django.db import connections
 from django.utils.functional import cached_property
 
-from pootle.core.delegate import subcommands
+from dj import subcommand
 
+from pootle.core.delegate import subcommands
 
-class SubcommandsError(CommandError):
 
+class SubcommandsError(subcommand.subcommands.SubcommandsError):
```

Comment:
Do we need to define here a exception that has been defined in third party package?

Reply:
i did this to avoid code changes elsewhere, but maybe it wasnt necessary - ill check